model.py

```diff
@@ -8610,6 +8609,25 @@ class Collection(Base):
     # external_account_id.
     parent_id = Column(Integer, ForeignKey('collections.id'), index=True)
 
+    # A dict capturing the column location of the unique collection
+    # account identifier in the collections table.
+    UNIQUE_IDENTIFIER_BY_PROTOCOL = {
+        # Axis 360 'username'
+        AXIS_360 : username,
```

Comment:
I think `external_account_id` (Axis "library ID") is a more reliable identifier of the collection. One user account in Axis 360 could conceivably have access to multiple libraries.